src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -321,6 +332,13 @@ public function viewAction(
             'isMultistoreEnabled' => $this->get('prestashop.adapter.feature.multistore')->isActive(),
             'transferGuestAccountForm' => $transferGuestAccountForm,
             'privateNoteForm' => $privateNoteForm->createView(),
+            'layoutTitle' => $this->trans(
+                'Customer %name%',
```

Comment:
```suggestion
                'Customer: %name%',
```

I really could use a separator there to make it more readable. It's a global suggestion. Wdyt?

Reply:
It's a good idea but if we do it here, then we have to do it everywhere.